langchain/scripts/create-entrypoints.js

```diff
@@ -360,6 +361,7 @@ const requiresOptionalDependency = [
   "tools/google_calendar",
   "tools/gmail",
   "callbacks/handlers/llmonitor",
+  "callbacks/handlers/lunary",
```

Comment:
We've deprecated new integration exports from `langchain` - can you remove and change examples to import from `@langchain/community/callbacks/handlers/lunary`?